jobs/build/advisories/Jenkinsfile

```diff
@@ -0,0 +1,212 @@
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: [
+                                'aos-team-art@redhat.com'
+                            ].join(',')
+                        ],
+                        [
+                            name: 'SPECIAL_NOTES',
+                            description: '(Optional) special notes to include in the email',
+                            $class: 'hudson.model.TextParameterDefinition',
+                            defaultValue: ""
+                        ],
+                    ]
+                ],
+            ]
+        )
+
+        commonlib.checkMock()
+
+        currentBuild.description = ""
+        try {
+            stage("kinit") {
+                buildlib = load("pipeline-scripts/buildlib.groovy")
+                buildlib.kinit()
+            }
+            stage("create rpm advisory") {
+                cmd = "-g openshift-${params.VERSION} "
+                cmd+= "create -k rpm "
+                cmd+= "--assigned-to ${params.ASSIGNED_TO} "
+                cmd+= "--manager ${params.MANAGER} "
+                cmd+= "--package-owner ${params.PACKAGE_OWNER} "
+                cmd+= "--impetus ${params.IMPETUS} "
+                cmd+= params.DATE ? "--date ${params.DATE} " : ""
+                cmd+= params.DRY_RUN ? "" : "--yes"
+
+                echo "elliott cmd: ${cmd}"
+                out = buildlib.elliott(cmd, [capture: true])
+                echo "out: ${out}"
+
+                rpm_advisory_id = buildlib.extractAdvisoryId(out)
+                echo "extracted rpm_advisory_id: ${rpm_advisory_id}"
+            }
+            stage("create image advisory") {
+                cmd = "-g openshift-${params.VERSION} "
+                cmd+= "create -k image "
+                cmd+= "--assigned-to ${params.ASSIGNED_TO} "
+                cmd+= "--manager ${params.MANAGER} "
+                cmd+= "--package-owner ${params.PACKAGE_OWNER} "
+                cmd+= "--impetus ${params.IMPETUS} "
+                cmd+= params.DATE ? "--date ${params.DATE} " : ""
+                cmd+= params.DRY_RUN ? "" : "--yes"
+
+                echo "elliott cmd: ${cmd}"
+                out = buildlib.elliott(cmd, [capture: true])
+                echo "out: ${out}"
+
+                image_advisory_id = buildlib.extractAdvisoryId(out)
+                echo "extracted image_advisory_id: ${image_advisory_id}"
+            }
+            stage("update job description with advisory links") {
+                currentBuild.description = params.DRY_RUN ? "[DRY_RUN]" : ""
+                currentBuild.description+= "https://errata.devel.redhat.com/advisory/${rpm_advisory_id}\n"
+                currentBuild.description+= "https://errata.devel.redhat.com/advisory/${image_advisory_id}"
+            }
+            stage("commit new advisories to ocp-build-data") {
+                cmd = "rm -rf ocp-build-data ; "
```

Comment:
And even more so here... it's an embedded shell script, why not make it look like one?